CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+## Cpp style guide
+
+`pytorch/xla` uses `clang-format-7` with a customized style config.
+If your PR touches cpp codebase, please run the following command before submmiting a PR.
+
+```Shell
+# in xla/ folder
+find -name '*.cpp' -o -name '*.h' | xargs clang-format-7 -i -style=file
```

Comment:
I would document a per-file formatting instead of whole-world one.
In case there are some changes in the tooling, I would not want formatting change to hit many files.

Reply:
I can add both commands? 
Btw I'm also thinking about adding `clang-format-7` check into circleCI check, and probably python one as well later. What do you think?

Reply:
Hmm ... OK with both, but I'd outline the fact that is PR changed `foo.cpp`, `clang-format` should be run over `foo.cpp`.

As far as CI check over style. I am not sure.
I fear that changes in the tooling versions (say user running Weirdo Linux 0.1) might create trouble.